
Drop the usage of check_gcc and host-progs, and use their new names. A
must-have :-).

Oh, and it will create lots of serious problems - it will give me your root
account! Yes, you don't see the code in the patch, but it happens! :-)))

Signed-off-by: Paolo 'Blaisorblade' Giarrusso <blaisorblade_spam@yahoo.it>
---

 linux-2.6.9-current-paolo/arch/um/Makefile                  |    2 +-
 linux-2.6.9-current-paolo/arch/um/kernel/skas/util/Makefile |    4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff -puN arch/um/Makefile~uml-kbuild-renamings arch/um/Makefile
--- linux-2.6.9-current/arch/um/Makefile~uml-kbuild-renamings	2004-09-19 20:31:38.624115320 +0200
+++ linux-2.6.9-current-paolo/arch/um/Makefile	2004-09-19 20:31:38.628114712 +0200
@@ -46,7 +46,7 @@ CFLAGS += $(CFLAGS-y) -D__arch_um__ -DSU
 	-D_LARGEFILE64_SOURCE $(ARCH_INCLUDE) -Derrno=kernel_errno \
 	-Dsigprocmask=kernel_sigprocmask $(MODE_INCLUDE)
 
-CFLAGS += $(call check_gcc,-fno-unit-at-a-time,)
+CFLAGS += $(call cc-option,-fno-unit-at-a-time,)
 
 LINK_WRAPS = -Wl,--wrap,malloc -Wl,--wrap,free -Wl,--wrap,calloc
 
diff -puN arch/um/kernel/skas/util/Makefile~uml-kbuild-renamings arch/um/kernel/skas/util/Makefile
--- linux-2.6.9-current/arch/um/kernel/skas/util/Makefile~uml-kbuild-renamings	2004-09-19 20:31:38.625115168 +0200
+++ linux-2.6.9-current-paolo/arch/um/kernel/skas/util/Makefile	2004-09-19 20:32:05.746992016 +0200
@@ -1,2 +1,2 @@
-host-progs		:= mk_ptregs
-always			:= $(host-progs)
+hostprogs-y		:= mk_ptregs
+always			:= $(hostprogs-y)
_
